### PR TITLE
EC-374 MVP - InfluxActor

### PR DESCRIPTION
**Pull request by GusevTimofey**

### CODE REVIEW

File: src/main/resources/application.conf [hunk 1]
```diff
@@ -3,6 +3,11 @@ thisNode {
   port = 8888
 }
 
+mvpStat {
+  nodeName = "mvp"
```
Comment (kapinuss) [inline src/main/resources/application.conf]: better send stats by ip

File: src/main/scala/mvp/MVP.scala [hunk 2]
```diff
@@ -17,6 +18,7 @@ object MVP extends App {
 
   system.actorOf(Props[Starter], "starter")
   system.actorSelection("/user/starter") ! Start
+  if (settings.mvpStat.sendStat) system.actorOf(Props[InfluxActor], "influxActor")
```
Comment (kapinuss) [inline src/main/scala/mvp/MVP.scala]: this actor should be a child of starter and be defined in bornKids()

File: src/main/scala/mvp/stats/InfluxActor.scala [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package mvp.stats
+
+import akka.actor.Actor
+import org.influxdb.{InfluxDB, InfluxDBFactory}
+import mvp.MVP.settings
+import mvp.stats.InfluxActor.CurrentBlockHeight
+
+class InfluxActor extends Actor {
+
+  val influxDB: InfluxDB = InfluxDBFactory.connect(settings.influxDB.url, settings.influxDB.login, settings.influxDB.password)
```
Comment (kapinuss) [inline src/main/scala/mvp/stats/InfluxActor.scala]: use /n in case of long string

File: src/main/scala/mvp/actors/Starter.scala [hunk 4]
```diff
@@ -23,6 +25,7 @@ class Starter extends Actor with StrictLogging {
     case Start if !settings.testMode => logger.info("real life baby on starter")
     case Heartbeat =>
       logger.info("heartbeat pong")
+      context.actorSelection("/user/starter/influxActor") ! CurrentBlockHeight()
```
Comment (Bromel777) [inline src/main/scala/mvp/actors/Starter.scala]: Add condition [if (settings.mvpStat.sendStat)]

File: src/main/scala/mvp/stats/InfluxActor.scala [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+class InfluxActor extends Actor with StrictLogging {
+
+  val influxDB: InfluxDB =
+    InfluxDBFactory.connect(settings.influxDB.url, settings.influxDB.login, settings.influxDB.password)
+
+  influxDB.setRetentionPolicy("autogen")
+
+  override def preStart(): Unit =
+    influxDB.write(8089, s"""nodesStartTime value="${settings.mvpStat.nodeName}"""")
```
Comment (Bromel777) [inline src/main/scala/mvp/stats/InfluxActor.scala]: nodeStartTime

**Pull request merged**